network/p2p/conduit/conduit.go

```diff
@@ -5,18 +5,33 @@ import (
 	"fmt"
 
 	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/component"
+	"github.com/onflow/flow-go/module/irrecoverable"
 	"github.com/onflow/flow-go/network"
 )
 
 // DefaultConduitFactory is a wrapper around the network Adapter.
 // It directly passes the incoming messages to the corresponding methods of the
 // network Adapter.
 type DefaultConduitFactory struct {
+	component.Component
+	cm      *component.ComponentManager
```

Comment:
```suggestion
	*component.ComponentManager
```
Directly embedding `ComponentManager` will make `DefaultConduitFactory` implement `component.Component`  without having two fields for the same object -- is that why you have `component.Component` embedded currently?

Reply:
Fixed in https://github.com/onflow/flow-go/pull/2308/commits/43fb4c956c72a35ef26aeaf9ce3801bfa439b29b